lib/linter-views.coffee

```diff
@@ -131,75 +132,47 @@ class LinterViews
       )
       throw null
 
-  updateCurrentLine: (line) ->
-    if @currentLine isnt line
-      @currentLine = line
-      @updateLineMessages()
-      @renderPanel()
-
-
-  updateLineMessages: ->
-    activeEditor = atom.workspace.getActiveTextEditor()
-    @linter.eachEditorLinter (editorLinter) =>
-      return unless editorLinter.editor is activeEditor
-
-      @lineMessages = []
+  updateLineMessages: (shouldRender = false) ->
+    return unless @tabs['Line'].visibility
+    @lineMessages.clear()
+    activeTextEditor = atom.workspace.getActiveTextEditor()
```

Comment:
Sorry, I thought after 50 commits this seemed like it was done.

I still think you should open another PR instead of pushing directly to master.